applications/models.py

```diff
@@ -53,6 +53,18 @@ def year_not_in_future(year: Optional[int]):
         raise ValidationError(format_lazy("{year} {msg}", year=year, msg=msg))
 
 
+class StatusBaseClass:
```

Comment:
I would leave `Class` from the name => `StatusBase` or would call it a mixin class => `StatusMixin` :thinking:

Reply:
That's a good point! I'll change it to `StatusMixin` as I feel that is more descriptive than Base.